adjust splittable-num-banks to reduce job runtimes by a factor of 4

couvares opened the pull request:
@duncan-brown, @bbockelm, and I observed in the OSG accounting data that we have lots of jobs running for >12 hours, beyond which we see a big uptick in badput (aka job re-runs due to preemption).  This patch reduces the job runtime by a factor of 4 in order to reduce badput without making the average job runtime too short.

This should result in a substantial (10-20%?) increase in overall workflow throughput, and possibly a reduction in total workflow wall-clock time (aka time to result).

couvares committed:
adjust splittable-num-banks to reduce job runtimes by a factor of 4

After observing in the OSG accounting data that we have lots of jobs running for >12 hours, beyond which we see a big uptick in badput (aka job re-runs due to preemption), this reduces the job runtime by a factor of 4 in order to reduce badput without making the average job runtime too short.

bbockelm commented:
For reference, here's the histogram of runtimes of successful jobs:

![image](https://user-images.githubusercontent.com/1093447/29336354-7d8514a2-81db-11e7-9212-7f4600a4bbc8.png)

The jobs are binned by hour-wide bins and the height of the bar is the sum of the core-hours for jobs in that bin.  It's not shown on this graph but, past 8-hours, there is a significant increase jobs that get preempted & restarted on OSG.  This change ought to provide a reasonable boost in throughput.

bbockelm reviewed (approved):
Looks fine to me.  We should revisit after the next run to determine if we're happy with the resulting job distribution.

couvares committed:
correction to splittable-num-banks for workflow-splittable-full_data

I accidentally increased splittable-num-banks for workflow-splittable-full_data by only a factor of 2, not 4.  My intent (and the estimated runtimes I quote in my comment) was a factor of 4.

spxiwh commented:
I'm concerned that this change will achieve the opposite effect when running on atlas or vulcan (these numbers were empirically tuned for good throughput when running there). We should maybe consider different setup options for different locations.

duncan-brown commented:
@spxiwh this shouldn't have any effect on atlas or vulcan, as it does not significantly increase the number of short running jobs.
